unconference/volunteer-support-guide.md

```diff
@@ -0,0 +1,118 @@
+# Organizational Guide for Supporting Volunteers and Reducing Burnout
+
+```{article-info}
+:avatar: images/disc-icon.jpg
+:avatar-outline: muted
+:author: "[Abigail Dogbe](https://github.com/mesrenyamedogbe), [Nicole Foster](https://github.com/nicole-numfocus),
+[Hana Frluckaj](https://github.com/hfrlucka), [Inessa Pawson](https://github.com/inessapawson), [Leah Silen](https://github.com/lsilen)"
+:class-container: sd-p-2 sd-outline-muted sd-rounded-1
+```
+
+The authors of this guide are connected with open source communities to varying degrees, but recommendations and lessons from this guide 
+can be broadly applicable to many types of organizations who interact frequently with volunteers. This guide was produced during the 
+DISC Unconference in Amsterdam, NL, 9-10 Sep 2023. 
+
+# Introduction
+Volunteering is a critical form of support for many organizations. There are many benefits and opportunities available to volunteers 
+through the nature of their work, yet volunteer burnout is too common of a phenomenon. This comprehensive guide is meant to help 
+organizations reduce volunteer burnout in two ways, by:
+recognizing the many barriers to and stressors of volunteering
+providing actionable suggestions for organizations to combat these stressors and support volunteers
+
+# Audience
+This guide is for organizations and communities that work with volunteers and individuals and groups who lead and interact with volunteers. 
+
+#Purpose
+The content of this guide can be adapted by individual organizations to help them design volunteer 
+management policies and procedures to reduce burnout and improve the experience of their volunteers.     
+
+# Definitions
+*Volunteer* - someone who performs a service or work voluntarily, usually unpaid 
+*Burnout* - emotional or physical exhaustion, usually as a response to chronic emotional and interpersonal stressors on the job
+*Marginalized groups* - groups who historically have been underserved and undervalued, largely due to identity demographics
+(e.g. people who are BIPOC, women, gender minorities, etc.) 
+
+# Problem Statement
+People get involved in volunteering for a number of reasons: as a form of self-education or skill-building, to network and 
+build community, to gain access to open spaces, etc. It is often an entry point into open source and future opportunities. 
+
+The open nature of volunteerism paradoxically can also reproduce societal inequalities. Volunteer works cuts into leisure 
+time and inherently demands stability, resources, energy and time from volunteers – therein lies the source of many issues 
+related to burnout, as volunteering becomes more unsustainable over time. Volunteer participation is dependent on an 
```

Comment:
If "volunteering becomes more unsustainable over time" is always true, thought I'm not sure that it is, perhaps a way of thinking about this would be how to extend a volunteer's period of effort.